### DISCUSSION TITLE
How do I link from the DB to external sources?

### DISCUSSION THREAD

**Discussion by indiancoffeeguy** [Budibase/budibase#1120] [Help]
Hi,
I am trying to create a content feed for users. While I have managed to create the page, I am unable to link the rows to external sources. 
Please suggest.

**Comment by joebudi** [comment]
Hey @indiancoffeeguy to clarify, you have created a table in the data section, generated a screen, and would now like to link the rows of that table to external sources, outside your app?

**Comment by indiancoffeeguy** [comment]
Yes. That’s correct.

**Reply by joebudi** [reply]
With this answer, I am assuming you have a column in your data that contains the external source link.

If this is the case, in the frontend, I would use a repeater + card, instead of the grid/table provided. 
The card will have a link. Bind the card's link destination field to your external source column.

That should work :-)

**Comment by indiancoffeeguy** [comment]
Hey Joe,
I've tried it already. The Link URL repeater + Card(Stacked List) allows me
to only link to pages on the site. Here's the screenshot.
[image: Screenshot 2021-02-20 at 10.02.13 AM.png]

**Comment by indiancoffeeguy** [comment]
FYI...I am using Text as the type in the DB for the URL.
